protocol/avdl/keybase1/ephemeral.avdl

```diff
@@ -0,0 +1,14 @@
+@namespace("keybase.1")
+
+protocol ephemeral {
+  import idl "common.avdl";
+
+  record DeviceEkMetadata {
```

Comment:
nit but let's call this `DeviceEKMetadata`

Reply:
I tried that and some LintVetCheckerHookBotAutomatonFactory complained :(